docs/matchers/parse_format.md

```diff
@@ -0,0 +1,163 @@
+# Parse Format Matcher
+
+## Configuring opsdroid
+
+As in [regex matcher](/matchers/regex.md), in order to enable parse format skills, you must set the `enabled` parameter to true in the parsers section of the opsdroid configuration file.
+
+```yaml
+parsers:
+  - name: parse_format
+    enabled: true
+```
+
+## About Parse Format Matcher
+
+This is the simplest matcher available in opsdroid. It matches the message from the user against a string with python [format syntax](https://docs.python.org/3/library/string.html#format-string-syntax). If the string matches then the function is called.
+
+The third party library [parse](https://github.com/r1chardj0n3s/parse) is used to parse the strings.
+
+You can specify which kind of matching you want to apply through the `matching_condition` kwarg (*match* matching is the default). Be careful, the matching condition names are the same than in regex but there are some differences between them.
+
+Matching conditions:
+
+- **match** - Scans through the string looking at the beginning of the string to match the pattern, **but the end must match too**. So if you have `hi` it will match `hi`, but no `say hi` or `hi!`.
+- **search** - Scans through the string looking for the first location where the pattern produces a match, and ignores what is before or after the match. So if you have `hi` it will match `hi`, `say hi` or `hi!`.
+
+
+### (Very) Basic example
+
+```python
+from opsdroid.skill import Skill
+from opsdroid.matchers import match_parse
+import random
+
+class MySkill(Skill):
+    @match_parse('hi')
+    async def hello(self, message):
+        text = random.choice(['Hi {}', 'Hello {}', 'Hey {}']).format(message.user)
+        await message.respond(text)
+```
+
+When we run opsdroid and send the message *"hi"* opsdroid will match the hello to the skill and reply immediately. Opsdroid will only trigger the skill if *hi* is only word in the message.
```

Comment:
Maybe add `the` before "only word" in the message. So the text will be:  `if *hi* is the only word in the message`